src/uwtools/resources/rocoto.jsonschema

```diff
@@ -1,44 +1,55 @@
 {
-    "attrs": {
-        "realtime": {
+    "workflow": {
+        "attrs": {
+            "realtime": {
+                "type": "string",
+                "enum": [
+                    "true",
+                    "false",
+                    "T",
+                    "F",
+                    "t",
+                    "f",
+                    "True",
+                    "False",
+                    "FALSE",
+                    "TRUE"
+                ]
+            }
+            "scheduler": {
+                "type": "string",
+                "enum": [
+                    "sge",
+                    "lsf",
+                    "lsfcray",
+                    "ll",
+                    "torque",
+                    "moabtorque",
+                    "moab",
+                    "pbspro",
+                    "slurm",
+                    "cobalt"
+                ]
+            }
+            "cycylethrottle": {
+                "type": "integer",
+                "minimum": 0
+            }
+            "taskthrottle": {
+                "type": "integer",
+                "minimum": 0
+            }
+        }
+        "cycledefs": {
             "type": "string",
-            "enum": [
-                "true",
-                "false",
-                "T",
-                "F",
-                "t",
-                "f",
-                "True",
-                "False",
-                "FALSE",
-                "TRUE"
-            ]
+            "pattern": "^\[\d{10},\d{10},(0[0-9]|1[0-9]|2[0-4]):([0-5][0-9]):([0-5][0-9])\]$"
         }
-        "scheduler": {
+        "log": {
             "type": "string",
-            "enum": [
-                "sge",
-                "lsf",
-                "lsfcray",
-                "ll",
-                "torque",
-                "moabtorque",
-                "moab",
-                "pbspro",
-                "slurm",
-                "cobalt"
-            ]
+            "format", "uri"
+        }
```

Comment:
I'm confused about `log`. The Jinja2 template has
```
<log>{{ log }}</log>
```
which suggests to me that we'd supply something like a path string here (maybe involving an entity reference), but the realized XML from S3 shows
```
<log><cyclestr>/scratch1/BMC/zrtrr/uw/srw/expt_dirs/grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2/log/FV3LAM_wflow.log</cyclestr></log>
```
So I'm curious where the nested `<cyclestr>` comes from...

Oh, looking at the RelaxNG schema, it looks like the value of `<log>` can either be just text, or a `compoundTimeString`, which provides the `<cyclestr>` nested element. @christinaholtNOAA given that the canned SRW case uses this second form, but our Jinja2 template seems not to support this, what's the requirement?

Reply:
The really challenging part of this thing is that a cyclestr can show up almost anywhere that you have a string, so it felt really tough to code into the Jinja2 template. I did have support for it in an earlier version with an elementtree, though.

Reply:
I would really not like to see an XML literal like `<cyclestr>` appear in UW YAML. If we can detect the condition(s) under which it needs to appear, having Python code insert that tag might be the least-bad interim solution while we're generating XML with Jinja2.

Reply:
I'd be fine with that, for sure.

So if we found anything like @Y@m@d@H in the string, we'd need it, but then we still don't have a great way to treat the offset attribute.

Reply:
Added a new Jira ticket to handle the `<cyclestr>` tag, [linked here](https://jira.epic.oarcloud.noaa.gov/browse/UW-386)